js/client/modules/@arangodb/testsuites/arangobench.js

```diff
@@ -201,67 +231,85 @@ function arangobench (options) {
 
       let args = _.clone(benchTodo);
       delete args.transaction;
-
+      delete args['expected-failure'];
       if (options.hasOwnProperty('benchargs')) {
         args = Object.assign(args, options.benchargs);
       }
-
       let oneResult = pu.run.arangoBenchmark(options, instanceInfo, args, instanceInfo.rootDir, options.coreCheck);
-      print();
-
-      if (benchTodo.hasOwnProperty('duration')) {
-        oneResult.status = oneResult.status && oneResult.duration >= benchTodo['duration'];
-        if (!oneResult.status) {
-          oneResult.message += ` didn't run for the expected time ${benchTodo.duration} but only ${oneResult.duration}`;
+      if (benchTodo.hasOwnProperty('expected-failure')) {
```

Comment:
This only checks if `expected-failure` is present, but not if it is set to `true` or `false`. This is a bit counter-intutitive, e.g. if someone else adds a test and sets the flag to `false`.

Reply:
Addressed